Project.toml

```diff
@@ -11,6 +11,16 @@ RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[weakdeps]
+Juno = "e5e0dc1b-0480-54bc-9374-aad01c23163d"
+SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+
+[extensions]
+MeasurementsJunoExt = "Juno"
+MeasurementsSpecialFunctionsExt = "SpecialFunctions"
+MeasurementsUnitfulExt = "Unitful"
+
 [compat]
 Calculus = "0.4.1, 0.5"
 RecipesBase = "0.6.0, 0.7, 0.8, 1.0"
```

Comment:
Also [`src/plot-recipes.jl`](https://github.com/JuliaPhysics/Measurements.jl/blob/f1d6e89b4ef6f24f5cb710c4f71d35afabf76af0/src/plot-recipes.jl) should be an extension.